pages/en/lb4/Creating-components.md

```diff
@@ -228,6 +230,33 @@ export class AuthenticationProvider {
 }
 ```
 
+## Extends Application
+
+When binding a component to an app, you may want to mount the component's properties 
+and methods to application level context. 
+A strategy to do that is using mixin, which extends a class with new properties and methods.
+
+An example scenario would be an app has multiple components with repositories bound to each of them, 
+you can use a `RepositoryMixin` to mount all of the repositories to the app.
+
+If you are not familiar with concept `mixin`, check [Mixin](Mixin.htm) to know more, 
+it also introduces how to apply mixins by the shortcut function in LoopBack.
+
+From Janny:
+
+I am going to add a Lite version of `RepositoryMixin`'s code as the end of this section
```

Comment:
Here I think you have some options. You could:
 - Give an overview of `RepositoryMixin` and point to its implementation
 - Give a brief overview of `RepositoryMixin` with code snippets like you said, and point to the source code at the end

I think the first option might suffice, given that Mixins are described in detail in their own page. IMO, the main points this section should cover would be:
- How component developers can leverage mixins
- Guideline on how they should use mixins 
- other considerations that they should take into account